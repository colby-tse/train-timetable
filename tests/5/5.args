Sydney Adelaide "12:02:00"
