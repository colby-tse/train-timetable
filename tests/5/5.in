Sydney::Adelaide::09:03:10
Sydney::Adelaide::10:23:35
Sydney::Adelaide:::14:54:22
Sydney::Adelaide::18:21:09
