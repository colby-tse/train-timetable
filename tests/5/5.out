The next train to Adelaide from Sydney departs at 18:21:09
